### DISCUSSION TITLE
استفاده از دو فالبک مادر

### DISCUSSION THREAD

**Discussion by farshadl** [Gozargah/Marzban#1014] [Q&A]
سلام. من روی ۲ تا ساب دامنه متفاوت با سرتیفیکیت‌های متفاوت می‌خوام فالبک بزنم. بنابراین باید دو تا فالیک مادر بسازم. برای این منظور تو تنظیمات مرزبان باید۲ خط مثل ۲ خط زیر اضافه کنم؟ لطفا یکی راهنمایی کنه
XRAY_FALLBACKS_INBOUND_TAG = "FALLBACK_INBOUND_01"
XRAY_FALLBACKS_INBOUND_TAG = "FALLBACK_INBOUND_02"

**Comment by omid-sadeghi** [comment]
از مستندات مرزبان و همچنین محتوای فایل env اینطور به نظر میاد که فقط میشه یک فالبک توی فایل env تعریف کرد هر چند که به صراحت جایی گفته نشده. اما چون فالبک یک قابلیت بنیادی xray و ربطی به پنل نداره، قاعدتا میشه چندین کانفیگ فالبک ساخت. الان خودم تست کردم، فالبک دوم رو ساختم و کار کرد و داخل env هم یک XRAY_FALLBACKS_INBOUND_TAG تعریف کردم.
